src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -253,7 +253,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         Friend Overrides Function GetImports() As ImmutableArray(Of String)
             ' TODO: implement (only called from VBI) https://github.com/dotnet/roslyn/issues/5854
```

Comment:
Oops, I just asked you to re-add this.  @AlekseyTs, the comment is still relevant because full import handling is more complicated - this is just a first step.